.github/workflows/release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       next-release-tag: ${{ steps.prepare-release.outputs.next-release-tag }}
     steps:
       # Check out the repo with credentials that can bypass branch protection, and fetch git history instead of just latest commit
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.2
```

Comment:
these are github native - so this doesn't really need to happen but still a good idea